src/test/java/gov/healthit/chpl/aqa/stepDefinitions/CHPLResourcesDropdownSteps.java

```diff
@@ -0,0 +1,35 @@
+package gov.healthit.chpl.aqa.stepDefinitions;
+import static org.junit.Assert.assertEquals;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+import org.openqa.selenium.support.ui.WebDriverWait;
+import cucumber.api.java.en.Then;
+import cucumber.api.java.en.When;
+import gov.healthit.chpl.aqa.pageObjects.ChplResourcesDropdownPage;
+
+/**
+ * Class CHPLResourcesDropdownSteps definition.
+ */
+public class CHPLResourcesDropdownSteps extends BaseSteps {
+    /**
+     * Click CHPL Resources Top Navigation.
+     */
+
+    @When("^I click CHPL Resources Top Navigation Link$")
+    public void iclickCHPLResourcesTopNavigationLink() {
```

Comment:
I think this should move to the BaseSteps file as well, as this is usable across 95% of the pages

Reply:
If, for some reason, we had to get rid of the ResourcesDownload page, the step about navigation to the Resources dropdown would disappear, because the "When" annotation isn't in the BaseSteps class. This entire function should go away, and the @When annotation should be in the BaseSteps file